sympy/utilities/tests/test_lambdify.py

```diff
@@ -119,6 +119,20 @@ def test_mpmath_lambda():
 
 
 @conserve_mpmath_dps
+def test_issue_12173_1():
+    mpmath.mp.dps = 50
+    exp= lambdify((x,y), uppergamma(x,y))(1,2)
+    assert exp==mpmath.mpf('0.13533528323661270231781372785917483270168304443359')
+
+
+@conserve_mpmath_dps
+def test_issue_12173_2():
```

Comment:
Both uppergamma and lowergamma can be tested in the same test_issue_12173. There is no need to define another test function.